### PR TITLE
📊✨ Show equalizer icon in stories

### PR DESCRIPTION
**Pull request by alanorozco**

### CODE REVIEW

File: src/service/video-manager-impl.js [hunk 1]
```diff
@@ -692,83 +692,84 @@ class VideoEntry {
...
+      if (mask) {
+        removeElement(mask);
+      }
+    });
 
-    function adStart() {
-      setStyles(mask, {
-        'display': 'none',
-      });
+    if (!this.video.isInteractive()) {
```
Comment (aghassemi) [inline src/service/video-manager-impl.js]: how does this end up being `true` in stories without `control` ?  My understanding is Stories wants this icon to show up but not use `controls` attribute on videos.

Comment (alanorozco) [reply src/service/video-manager-impl.js]: This changes the semantics of autoplay somewhat, the equalizer icon will be displayed for all videos including those without `controls`. I figured by having the `noaudio` flag we're covered?

Comment (aghassemi) [reply src/service/video-manager-impl.js]: that would be a breaking change for sites like bmw.com. The idea was if 'controls' is not specified, the `video` is essentially a background design creative and equalizer icon should not show up.

File: extensions/amp-story/1.0/amp-story.css [hunk 2]
```diff
@@ -526,6 +526,12 @@ amp-story[standalone] .i-amphtml-story-developer-log {
   display: none !important;
 }
 
+amp-story .amp-video-eq {
+  /* Align with spinner */
```
Comment (newmuis) [inline extensions/amp-story/1.0/amp-story.css]: Maybe actually use the same rules for both?

File: src/service/video-manager-impl.js (hunk 1 above)
Comment (newmuis) [reply src/service/video-manager-impl.js]: Maybe we can explicitly expose an attribute to show the equalizer?

Comment (alanorozco) [reply src/service/video-manager-impl.js]: @newmuis I think making a CSS exception for stories i.e.

```csss
.amp-video-eq { display: none }
[controls] .amp-video-eq,

File: css/video-autoplay.css [hunk 3]
```diff
@@ -22,17 +22,21 @@ i-amphtml-video-mask {
  * Animated equalizer icon for muted autoplaying videos.
  */
 
-[noaudio] .amp-video-eq {
+.amp-video-eq {
   /* Don't display icon for videos flagged as having no audio. */
```
Comment (newmuis) [inline css/video-autoplay.css]: Shouldn't this comment be updated?  It's more than just `noaudio` videos that won't have the equalizer.

Comment (alanorozco) [reply css/video-autoplay.css]: Done! And it helped me realize that The `noaudio` case was broken, so fixed. 😅

**Pull request merged**